newrelic/resource_newrelic_cloud_gcp_integrations.go

```diff
@@ -19,201 +19,206 @@ func resourceNewrelicCloudGcpIntegrations() *schema.Resource {
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsFunctionsSchemaElem(),
-				MaxItems:    1,
-			},
-			"interconnect": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsInterconnectSchemaElem(),
-				MaxItems:    1,
-			},
-			"kubernetes": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsKubernetesSchemaElem(),
-				MaxItems:    1,
-			},
-			"load_balancing": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsLoadBalancingSchemaElem(),
-				MaxItems:    1,
-			},
-			"mem_cache": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsMemCacheSchemaElem(),
-				MaxItems:    1,
-			},
-			"pub_sub": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsPubSubSchemaElem(),
-				MaxItems:    1,
-			},
-			"redis": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsRedisSchemaElem(),
-				MaxItems:    1,
-			},
-			"router": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsRouterSchemaElem(),
-				MaxItems:    1,
-			},
-			"run": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsRunSchemaElem(),
-				MaxItems:    1,
-			},
-			"spanner": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsSpannerSchemaElem(),
-				MaxItems:    1,
-			},
-			"sql": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsSQLSchemaElem(),
-				MaxItems:    1,
-			},
-			"storage": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsStorageSchemaElem(),
-				MaxItems:    1,
-			},
-			"virtual_machines": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsVirtualMachinesSchemaElem(),
-				MaxItems:    1,
-			},
-			"vpc_access": {
-				Type:        schema.TypeList,
-				Description: "GCP big table service",
-				Optional:    true,
-				Elem:        cloudGcpIntegrationsVpcAccessSchemaElem(),
-				MaxItems:    1,
-			},
+		Schema: generateGcpIntegrationSchema(),
+	}
+}
+
+func generateGcpIntegrationSchema() map[string]*schema.Schema {
+	baseSchema := cloudGcpIntegrationSchemaBase()
```

Comment:
```suggestion
	gcp_integrations_base_schema := cloudGcpIntegrationSchemaBase()
```

(just trying to correct the case, as generally used in Golang :) please rename this variable)

Reply:
Thanks for the suggestion, got it, I renamed the variable(gcpIntegrationsBaseSchema)